unlock-app/src/components/content/DashboardContent.js

```diff
@@ -1,5 +1,5 @@
 import styled from 'styled-components'
-import React, { useContext, useState } from 'react'
+import React, { useContext, useState, useCallback } from 'react'
```

Comment:
Let's clean that up please 🧹

Reply:
@julien51 fixed, and removed other non-used vars.
anyway, we can think about removing `no-unused-vars` from the `rulesToIgnore` in `.eslintrc`